purchase_duplicate_check/models/purchase_order.py

```diff
@@ -0,0 +1,57 @@
+from odoo import models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    def _prepare_pending_orders_message(self, product_id):
+        """
+        Prepare pending order line message
+
+        :param product_id: product.product record id
+        :return str: message
+        """
+        message_parts = []
+        order_lines = self.env["purchase.order.line"].search(
+            [("product_id", "=", product_id), ("order_id", "in", self.ids)]
+        )
+        for line in order_lines:
+            order = line.order_id
+            order_href = (
+                f"<a href='/web#id={order.id}&model={order._name}'>{order.name}</a>"
+            )
+            type_ = order.state in ["draft", "sent"] and "RFQ" or "PO"
+            message_parts.append(
+                f"{type_}: {order_href} date: {order.create_date.date()} Qty: {line.product_qty}<br/>"  # noqa
```

Comment:
This will looks like `RFQ: PO001 date: 01/01/2025 Qty: 10.0`, it would be much better to be shown as `RFQ: PO001 orders 10.0 unit, to confirm by 01/01/2025` and `PO: PO001 orders 10.0 unit, expected to received by 01/01/2025`

Reply:
> This will looks like `RFQ: PO001 date: 01/01/2025 Qty: 10.0`, it would be much better to be shown as `RFQ: PO001 orders 10.0 unit, to confirm by 01/01/2025` and `PO: PO001 orders 10.0 unit, expected to received by 01/01/2025`

I think this is too much.

Reply:
then at least separate the components with `-` or `;`